### DISCUSSION TITLE
Types for `syslog-pro`

### DISCUSSION THREAD

**Discussion by lunaisnotaboy** [DefinitelyTyped/DefinitelyTyped#61672] [Request a new @types package] [answered]
Hi there-

I maintain a fork of the software Misskey which uses the package [`syslog-pro`](https://npmjs.com/package/syslog-pro) for logging practically everything. We're planning on replacing it in our fork, but for the time being we would really appreciate it if someone was willing to create some types for the package.

Thanks!

**Comment by BendingBender** [comment, marked as answer]
@lunaisnotaboy I just created some definitions: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/61729. I would be grateful if you could test them on your codebase.

As long as the PR is not meged, you can test them via [this approach](https://github.com/DefinitelyTyped/DefinitelyTyped#testing). Basically, you need to create a `types.d.ts` file somewhere in your project where your TS compiler will pick them up (usually in a `src` directory, depending on your setup) and wrap the whole definition with: `declare module "syslog-pro" { <paste types from PR here> }`.

**Reply by lunaisnotaboy** [reply]
Thanks! I'll test it out.

**Reply by lunaisnotaboy** [reply]
Yeah, it works! Thank you so much.

**Reply by BendingBender** [reply]
You're welcome.
